pkg/apis/pipeline/v1alpha1/pod.go

```diff
@@ -87,4 +87,8 @@ type PodTemplate struct {
 	// default.
 	// +optional
 	PriorityClassName *string `json:"priorityClassName,omitempty" protobuf:"bytes,7,opt,name=priorityClassName"`
+
+	// SchedulerName specifies the scheduler to be used to dispath the Pod
+	//+optional
```

Comment:
two minor nits:

`dispath` -> `dispatch`

`//+optional` -> `// +optional`

Reply:
I will make these changes. Thanks for checking thi out.